x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/find_rules_status_route.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Hapi from 'hapi';
+import { isFunction } from 'lodash/fp';
+
+import { DETECTION_ENGINE_RULES_URL } from '../../../../../common/constants';
+import { ServerFacade } from '../../../../types';
+import { findRulesStatusesSchema } from '../schemas/find_rules_statuses_schema';
+import {
+  FindRulesStatusesRequest,
+  IRuleSavedAttributesSavedObjectAttributes,
+} from '../../rules/types';
+import { ruleStatusSavedObjectType } from '../../rules/saved_object_mappings';
+
+const convertToSnakeCase = (obj: IRuleSavedAttributesSavedObjectAttributes) => {
+  return Object.keys(obj).reduce((acc, item) => {
+    const newKey = item
+      .split(/(?=[A-Z])/)
+      .join('_')
+      .toLowerCase();
+    return { [newKey]: obj[item], ...acc };
+  }, {});
+};
```

Comment:
I will accept this but we need to change this with a follow up PR possibly as from the discussion ticket on Kibana I think they said they didn't want us to do a looping replace.

For now, can we quickly replace this with this call:
https://lodash.com/docs/4.17.15#snakeCase

To lower our code surface area?

Reply:
Just pushed the update with the replacement using lodash. I agree. Maybe something like the `transformAlertToRule` function you wrote?

Reply:
Later yeah. We can use the lodash one for now to get this across the line